python/lsst/pex/config/history.py

```diff
@@ -252,14 +252,14 @@ def format(config, name=None, writeSourceLine=True, prefix="", verbose=False):
     fullname = f"{config._name}.{name}" if config._name is not None else name
     msg.append(_colorize(re.sub(r"^root\.", "", fullname), "NAME"))
     for value, output in outputs:
-        line = prefix + _colorize("%-*s" % (valueLength, value), "VALUE") + " "
+        line = prefix + _colorize(f"{value:<{valueLength}}", "VALUE") + " "
         for i, vt in enumerate(output):
             if writeSourceLine:
-                vt[0][0] = "%-*s" % (sourceLength, vt[0][0])
+                vt[0][0] = f"{vt[0][0]:<{sourceLength}}"
 
             output[i] = " ".join([_colorize(v, t) for v, t in vt])
 
-        line += ("\n%*s" % (valueLength + 1, "")).join(output)
+        line += ("\n" + f"{'':>{valueLength + 1}}").join(output)
```

Comment:
Any reason why the `\n` can't go inside the f-string?

Reply:
Nice catch! Moved it inside the f-string.